lib/cubits/settings/settings_cubit.dart

```diff
@@ -0,0 +1,21 @@
+import 'package:coffeecard/data/repositories/v1/account_repository.dart';
+import 'package:coffeecard/models/account/user.dart';
+import 'package:equatable/equatable.dart';
+import 'package:flutter_bloc/flutter_bloc.dart';
+
+part 'settings_state.dart';
+
+class SettingsCubit extends Cubit<SettingsState> {
+  final AccountRepository _accountRepository;
+
+  SettingsCubit(this._accountRepository) : super(const SettingsState());
+
+  Future<void> loadUser() async {
+    final user = User.fromDTO(await _accountRepository.getUser());
```

Comment:
Let the account repository take care of this mapping unless we have a reason to use the dto somewhere else :)

Reply:
`getUser` now returns `Future<Either<ApiError, User>>` so this has been fixed